python/arcticdb/version_store/_normalization.py

```diff
@@ -301,11 +293,21 @@ def _normalize_single_index(index, index_names, index_norm, dynamic_strings=None
             index_tz = get_timezone(index[0].tzinfo)
 
         if index_tz is not None:
-            index_norm.tz = index_tz
+            index_norm.tz = _ensure_str_timezone(index_tz)
 
         return index_names, ix_vals
 
 
+def _ensure_str_timezone(index_tz):
+    if is_utc(index_tz):
+        # Pandas started to treat UTC as a special case and give back a pytz object for it. We coerce it back to
```

Comment:
`pandas` also does the same for UTC tzinfo objects from other timezone libs such as `dateutil` (and [in later versions](https://github.com/jbrockmendel/pandas/blob/main/pandas/_libs/tslibs/timezones.pyx#L75), `datetime.timezones.UTC`)

```suggestion
        # Pandas started to treat UTC as a special case and give back the tzinfo object for it. We coerce it back to
```